code/datums/antagonists/antag_datum.dm

```diff
@@ -83,8 +89,9 @@
 		new_body.antag_datums = list()
 	new_body.antag_datums += src
 	owner.antag_datums -= src
-	owner = new_body
-	apply_innate_effects()
+	spawn(1) //Give the game time to sort out new minds, bodies...
```

Comment:
this shouldn't be needed, as it should only be called after the mind has been transferred to the new body